### PR TITLE
STY: x.__class__ to type(x) #batch-3

### PR DESCRIPTION
**Pull request by ShaharNaveh**
- [x] ref https://github.com/pandas-dev/pandas/pull/29816
- [x] tests added / passed
- [x] passes `black pandas`
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [ ] whatsnew entry

### CODE REVIEW

File: pandas/io/clipboard/__init__.py [hunk 1]
```diff
@@ -156,7 +156,7 @@ def copy_qt(text):
         cb = app.clipboard()
         cb.setText(text)
 
-    def paste_qt() -> str:
+    def paste_qt():
```
Comment (jreback) [inline pandas/io/clipboard/__init__.py]: these annotations got removed?

**Review by jreback — CHANGES_REQUESTED** [review]

File: pandas/io/clipboard/__init__.py (hunk 1 above)
Comment (MomIsBestFriend) [reply pandas/io/clipboard/__init__.py]: No, I just copied files without seeing that they changed.

File: pandas/core/internals/blocks.py [hunk 2]
```diff
@@ -257,11 +257,11 @@ def make_block_same_class(self, values, placement=None, ndim=None):
             placement = self.mgr_locs
         if ndim is None:
             ndim = self.ndim
-        return make_block(values, placement=placement, ndim=ndim, klass=self.__class__)
+        return make_block(values, placement=placement, ndim=ndim, klass=type(self))
 
     def __repr__(self) -> str:
         # don't want to print out all of the items here
-        name = pprint_thing(self.__class__.__name__)
+        name = pprint_thing(type(self).__name__)
```
Comment (jbrockmendel) [inline pandas/core/internals/blocks.py]: pprint_thing should be unnecessarry here.  `__name__` should always be a str, right?  this is likely a py2 holdover

File: pandas/core/internals/managers.py [hunk 3]
```diff
@@ -321,7 +321,7 @@ def __len__(self) -> int:
         return len(self.items)
 
     def __repr__(self) -> str:
-        output = pprint_thing(self.__class__.__name__)
+        output = pprint_thing(type(self).__name__)
```
Comment (jbrockmendel) [inline pandas/core/internals/managers.py]: same re pprint_thing

File: pandas/core/series.py [hunk 4]
```diff
@@ -256,9 +256,7 @@ def __init__(
             elif is_extension_array_dtype(data):
                 pass
             elif isinstance(data, (set, frozenset)):
-                raise TypeError(
-                    "{0!r} type is unordered".format(data.__class__.__name__)
-                )
+                raise TypeError("{0!r} type is unordered".format(type(data).__name__))
```
Comment (jbrockmendel) [inline pandas/core/series.py]: can you use an fstring here

File: pandas/core/series.py [hunk 5]
```diff
@@ -1573,7 +1571,7 @@ def to_string(
             raise AssertionError(
                 "result must be of type unicode, type"
                 " of result is {0!r}"
-                "".format(result.__class__.__name__)
+                "".format(type(result).__name__)
```
Comment (jbrockmendel) [inline pandas/core/series.py]: above can you update "unicode" to str

File: pandas/core/indexing.py [hunk 6]
```diff
@@ -228,7 +228,9 @@ def _validate_key(self, key, axis: int):
         raise AbstractMethodError(self)
 
     def _has_valid_tuple(self, key: Tuple):
```
Comment (jbrockmendel) [inline pandas/core/indexing.py]: might as well annotate while in the neighborhood

Comment (jbrockmendel) [reply pandas/core/indexing.py]: never mind, for a second github was only showing me this one piece of the diff

File: pandas/io/clipboard/__init__.py [hunk 7]
```diff
@@ -96,7 +96,7 @@ def _stringifyText(text) -> str:
     if not isinstance(text, acceptedTypes):
         raise PyperclipException(
             f"only str, int, float, and bool values"
-            f"can be copied to the clipboard, not {text.__class__.__name__}"
+            f"can be copied to the clipboard, not {type(text).__name__}"
```
Comment (jbrockmendel) [inline pandas/io/clipboard/__init__.py]: looks like there should be a space at the end of the previous line

File: pandas/core/indexing.py (hunk 6 above)
Comment (MomIsBestFriend) [reply pandas/core/indexing.py]: > might as well annotate while in the neighborhood

I don't see it returning anything, annotate it as ```-> None```?

Comment (jbrockmendel) [reply pandas/core/indexing.py]: pls ignore, i was looking at the wrong diff

Comment (MomIsBestFriend) [reply pandas/core/indexing.py]: I know, but I saw other functions that need to annotated.

**Review by WillAyd — APPROVED** [review]

**Comment by WillAyd** [comment]
Thanks!

**Pull request merged**